examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react';
+
+import { ChannelSort, LiteralStringForUnion, StreamChat } from 'stream-chat';
+import { Channel, ChannelList, Chat } from 'stream-chat-react';
+
+import { ChannelContainer } from '../ChannelContainer/ChannelContainer';
+import { SideDrawer } from '../SideDrawer/SideDrawer';
+import { SocialChannelList } from '../SocialChannelList/SocialChannelList';
+import { SocialChannelListHeader } from '../SocialChannelList/SocialChannelListHeader';
+import { SocialChannelPreview } from '../ChannelPreview/SocialChannelPreview';
+import { SocialMessageInput } from '../MessageInput/SocialMessageInput';
+
+import { useViewContext } from '../../contexts/ViewContext';
+
+import './ChatContainer.scss';
+
+const apiKey = process.env.REACT_APP_STREAM_KEY;
+const user = process.env.REACT_APP_USER_ID;
+const userImage = process.env.USER_IMAGE;
+const userToken = process.env.REACT_APP_USER_TOKEN;
+
```

Comment:
gonna need all the url params stuff from the current social app, including the weird shit i had to add for the android/ios teams